### PR TITLE
Review

### PR DESCRIPTION
**Pull request by dvdplm**
Tweaks to language and formatting.

NOTE: there are a few links in the text that go to pages I do not find in the repo. And mentions of forms that I do not know if they are live or not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-* **Scalability:** Topos provides an impressive degree of scalability. It can handle substantial computational loads without compromising performance, which provides an efficient mechanism to create and broadcast certificates.
-* **Privacy through zero-knowledge proofs:** In the context of Topos, ZKPs are used to prove that transactions have been validated and their outcomes computed according to the protocol while keeping sensitive transaction details hidden. This technology represents a significant advancement in maintaining user privacy while retaining the ability to transparently verify the transaction history.
+* **Scalability:** Topos provides an impressive degree of scalability and can handle substantial computational loads without compromising performance.
+* **Privacy through zero-knowledge proofs:** In the context of Topos, ZKPs are used to prove that transactions are valid and their outcomes computed according to the rules of their subnet, all while keeping sensitive transaction details hidden. This technology represents a significant advancement in maintaining user privacy while retaining the ability to transparently verify the transaction history.
```
[the remark this answers is not in the document]
Comment (CitMC) [reply README.md:12]: I would recommend going with ZKP, so keeping it as is.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Consequence: A permanent ban from any sort of public interaction within the comm
 
 ## Attribution
```
[the remark this answers is not in the document]
Comment (CitMC) [reply CODE_OF_CONDUCT.md:72]: After consulting with Lucy, we believe we should keep the code of conduct as it is more grant program-specific and explains the process of rectifying transgression.

**Review by CitMC — APPROVED** [review]
All changes can be merged into main now. Made some minor adjustments in the numbering in files - in case someone only looks at the raw version and to keep the document structure from legal.

**Review by coldice — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dvdplm) [reply CODE_OF_CONDUCT.md:72]: So now we have two CoCs, one for grants and one for the rest of the project? That's suboptimal imo.

**Comment by CitMC** [comment]
Definitely a fair point to open up a new PR against the PR branch - that one slipped my mind in the rush.
The reasoning behind changing the numbering was that after checking with Ibo and Lucy again, all documents from legal needed to have the numbering also in the raw as in the signed-off version from legal, and then it would have been inconsistent across the files.

**Pull request merged**